### DISCUSSION TITLE
Export date as dd/MM/yyyy after updating to latest version

### DISCUSSION THREAD

**Discussion by istisan** [JoshClose/CsvHelper#1753] [Q&A]
Hi! I update CsvHelper from version 15.0.5 to the latest 26.1.0 and I am facing a breaking change on this code:
csvWriter.Configuration.TypeConverterOptionsCache.GetOptions<DateTime>().Formats = new[] { "dd/MM/yyyy" };
how can I change it to continue exporting all my dates to the format  "dd/MM/yyyy" ?
Thank you in advance,
Marco

**Comment by JoshClose** [comment]
```cs
csvWriter.Context.TypeConverterOptionsCache...
```
